src/Components/PoseGallery/PoseCard/PoseParts/Picture.jsx

```diff
@@ -1,24 +1,24 @@
 import React from 'react';
-import styled from 'styled-components'
+import styled from 'styled-components';
 import LoadDisplay from '../../../UI/Loader';
 import {
-	Desktop,
-	PhonePortrait,
-	PhoneLandscape,
+  Desktop,
+  PhonePortrait,
+  PhoneLandscape,
 } from '../../../UI/DeviceRules';
 
 const PosePicture = ({ img }) => (
   <picture>
-  <source
-  srcSet={img.replace('https://res.cloudinary.com/dz2nxhscn/image/upload/v1514930940/', 'https://s3.us-east-2.amazonaws.com/yardenimages/WeAcro/')}
-  media={Desktop}
-				/>
-  <Img
-  src={img.replace('https://res.cloudinary.com/dz2nxhscn/image/upload/v1514930940/', 'https://s3.us-east-2.amazonaws.com/yardenimages/WeAcro/')}
-  alt="Loading..."
-  loader={LoadDisplay}
-				/>
-			</picture>
+    <source
+      srcSet={img.replace('https://res.cloudinary.com/dz2nxhscn/image/upload/v1514930940/', 'https://s3.us-east-2.amazonaws.com/yardenimages/WeAcro/')}
```

Comment:
What is `<picture>` and `<source>`? Also our database is clean now so no need to replace

Reply:
removed